python/http_ece/tests/test_ece.py

```diff
@@ -424,13 +442,13 @@ def _run(self, mode):
         if mode == 'encrypt':
             func = ece.encrypt
             local = 'sender'
-            remote = 'receiver'
+            # remote = 'receiver'
             inp = 'input'
             outp = 'encrypted'
         else:
             func = ece.decrypt
             local = 'receiver'
-            remote = 'sender'
+            # remote = 'sender'
             inp = 'encrypted'
```

Comment:
If we don't use it, just remove the line.

Reply:
fair enough. I was siding toward caution in case that needed to be specified somewhere, but yeah, easy enough to add back if we need to.